projects/README.md

```diff
@@ -74,6 +74,9 @@ _Task & models for chitchat with a given persona._
 
 ## Well-Behaved
 
+- **SaFeRDialogues: Taking Feedback Gracefully after Conversational Safety Failures** [[project]](https://parl.ai/projects/safer_dialogues/) [[paper]](https://arxiv.org/abs/2110.07518).
```

Comment:
i think it's "https://parl.ai/projects/saferdialogues"

Reply:
good catch! thanks!